src/core/model_config.proto

```diff
@@ -448,27 +448,63 @@ message ModelOptimizationPolicy
   //@@
   message ExecutionAccelerators
   {
-    //@@    .. cpp:var:: string gpu_execution_accelerator (repeated)
+    //@@
+    //@@  .. cpp:var:: message Accelerator
+    //@@
+    //@@     Specify the accelerator to be used to execute the model.
+    //@@     Accelerator with the same name may accept different parameters
+    //@@     depending on the backends.
+    //@@
+    message Accelerator
+    {
+      //@@    .. cpp:var:: string name
+      //@@
+      //@@       The name of the execution accelerator.
+      //@@
+      string name = 1;
+
+      //@@    .. cpp:var:: map<string, string> parameters
+      //@@
+      //@@       Additional paremeters used to configure the accelerator.
+      //@@
+      map<string, string> parameters = 2;
+    }
+
+    //@@    .. cpp:var:: Accelerator gpu_execution_accelerator (repeated)
     //@@
     //@@       The preferred execution provider to be used if the model instance
     //@@       is deployed on GPU. The order priority implies priority, i.e. the
     //@@       provider at the front has highest priority.
     //@@
-    //@@       For ONNX Runtime backend, possible value is "tensorrt".
+    //@@       For ONNX Runtime backend, possible value is "tensorrt" as name,
```

Comment:
ONNXruntime has no way to control the TRT parameters like TF? We should file a ticket with ONNXRuntime for that.

Reply:
DLIS-766 is the tracker ticket, will open an issue on ONNX Runtime later.